dir2/file1.txt

```diff
@@ -1 +1,2 @@
 This is file1.txt added by ghouse
+This is the new line that has been added.
```

Comment:
Please remove full top at the end of the second line

Reply:
I have updated my code and removed period

Reply:
Good job, Ghouse!